src/ape_networks/_cli.py

```diff
@@ -163,3 +203,44 @@ def _run(cli_ctx, provider: "SubprocessProvider"):
     else:
         provider.disconnect()
         cli_ctx.abort("Process already running.")
+
+
+@cli.command(short_help="Stop node processes")
+@ape_cli_context()
+@click.option("--pid", "process_ids", help="The PID of the process(es) to kill", multiple=True)
+@network_option(default=None)
+def kill(cli_ctx, process_ids, provider):
+    """
+    Stop node processes
+    """
+    if not cli_ctx.network_manager.running_nodes:
+        echo_rich_text("No running nodes found.")
+
+    elif processes_killed := cli_ctx.network_manager.kill_node_process(*process_ids):
```

Comment:
According to your demo output, if you don't provide any process_ids, it kills all of them? But if you provide 1 then it only kills 1? I think that might not be expected behavior from a user perspective

Reply:
OK, I could add a `--all` flag. It kinda seems like 99% of the time, you are trying to kill them all but I could be wrong.

Reply:
[7447e6d](https://github.com/ApeWorX/ape/pull/2545/commits/7447e6dfacbb9e784314022cae1cde7263d478a0)